rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -918,6 +921,7 @@ def _get_logs(
             argument_filters: dict[str, Any],
             from_block: int,
             to_block: int | Literal['latest'] = 'latest',
+            callback: LogCallback | None = None,
```

Comment:
callback is too generic. It's a callback that will run at each log event query iteration. So `log_iteration_callback`. Can also do `log_iteration_cb` as `_cb` suffix is commonly used for callbacks, and you also have the type.

Rename for all functions.